collections/_release_4_5/entry-platforms-apple-export-to-visionos.md

```diff
@@ -0,0 +1,26 @@
+---
+type: entry
+section: platforms
+subsection: apple
+rank: 1
+importance: 3
+anchor: export-to-visionos
+title: Export to visionOS
+text: |
+  You can now export for Apple’s XR platform, visionOS, making Godot projects compatible with the Apple Vision Pro.
+
+  Godot 4.5 marks the first step of visionOS’ native integration. For this, we would like to thank Apple's visionOS engineering team for their contributions adding support for their operating system to the Godot Engine.
+
+  This is kinda a big deal: this is the first new platform supported natively by Godot since the project was open-sourced!
```

Comment:
```suggestion
  This is a big deal: the first new platform supported natively by Godot since the project was open-sourced!
```

Reply:
Or maybe "milestone" instead of "big deal".

Reply:
I removed "this is" because of the two "this is" in quick succession, but I agree, it seems a little bare without it. I think your suggestion of "visionOS is the..." is perfect.